modules/@apostrophecms/widget-type/ui/apos/components/AposWidgetEditor.vue

```diff
@@ -13,7 +13,9 @@
         <template #bodyMain>
           <div class="apos-widget-editor__body">
             <AposSchema
-              :schema="schema" v-model="docInfo"
+              :schema="schema"
+              v-model="docFields"
+              ref="schema"
```

Comment:
This `ref` isn't used, right? Can we nix that before merge?

Reply:
It's used in the mixin

Reply:
Yep